templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>This would be a bmp file if taken from D drive. Maybe just say "take optical image"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>